openfisca_france/parameters/prestations_sociales/prestations_etat_de_sante/invalidite/pensions_invalidite/revalorisations_invalidite.yaml

```
@@ -0,0 +1,164 @@
+  2004-01-01:
+    value: 1.017
+  2005-01-01:
+    value: 1.02
+  2006-01-01:
+    value: 1.018
+  2007-01-01:
+    value: 1.018
+  2008-01-01:
+    value: 1.011
+  2008-09-01:
+    value: 1.008
+  2009-04-01:
+    value: 1.01
+  2010-04-01:
+    value: 1.009
+  2011-04-01:
+    value: 1.021
+  2012-04-01:
+    value: 1.021
+  2013-04-01:
+    value: 1.013
+  2014-04-01:
+    value: 1.006
+  2015-04-01:
+    value: 1
+  2016-04-01:
+    value: 1.001
+  2017-04-01:
+    value: 1.003
+  2018-04-01:
+    value: 1.01
+  2019-04-01:
+    value: 1.003
+  2020-04-01:
+    value: 1.009
+  2021-04-01:
+    value: 1.001
+  2022-04-01:
+    value: 1.018
+  2022-07-01:
+    value: 1.04
+  2023-04-01:
+    value: 1.015385
+  2024-04-01:
+    value: 1.046
+metadata:
+  short_label: Indexation des pensions d'invalidité
+  label_en: Disability pension indexation
+  short_label_en: Disability pension indexation
+  ipp_csv_id: revalorg
+  reference:
+    1998-01-01:
+      title: Arrêté du 29/12/1997
+      href: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000387479/
+    1999-01-01:
+      title: Arrêté du 29/12/1998
+      href: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000757782/
+    2000-01-01:
+      title: Arrêté du 29/12/1999
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000762866
+    2001-01-01:
+      title: Arrêté du 26/12/2000
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000588355
+    2002-01-01:
+      title: Arrêté du 25/01/2002
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000399330
+    2003-01-01:
+      title: Arrêté du 31/12/2002
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000776880
+    2004-01-01:
+      title: Arrêté du 23/12/2003
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000776880
+    2005-01-01:
+      title: Arrêté du 16/12/2004
+      href: https://www.legifrance.gouv.fr/loda/id/LEGITEXT000005945891/
+    2006-01-01:
+      title: Arrêté du 23/12/2005
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000456325
+    2007-01-01:
+      title: Arrêté du 19/12/2006
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000820419
+    2008-01-01:
+      title: Arrêté du 21/12/2007
+      href: https://www.legifrance.gouv.fr/loda/id/LEGITEXT000017770723/
+    2008-09-01:
+      title: Lettre ministérielle du 31/07/2008
+      href: https://www.cdc.retraites.fr/IMG/html/lettre_ministerielle_du_31-07-2008.html
+    2009-04-01:
+      title: Lettre ministérielle du 25/03/2009
+      href: https://retraitesolidarite.caissedesdepots.fr/sites/default/files/fspoeie/lettre_ministerielle_du_25-mars-2009.pdf
+    2010-04-01:
+      title: Lettre ministérielle 2010/95 du 24/03/2010
+      href: https://www.legislation.cnav.fr/Pages/texte.aspx?ID=60684
+    2011-04-01:
+      title: Circ. interministérielle DSS/3A/2011/108 du 29/03/2011
+      href: https://www.legislation.cnav.fr/Pages/texte.aspx?Nom=CR_CN_2011030_14042011#6
```

Les liens 2009, 2010 et 2011, 2013 ne fonctionnent plus
```suggestion
      href: https://politiques-sociales.caissedesdepots.fr/sites/default/files/circulaire_interministerielle_du_29-mars-2011.pdf
```

---

Je n'ai pas retrouvé de liens pour 2009 et 2010 (et celui de 2013 est douteux).